runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -590,6 +592,22 @@ public void loadAppFromManifest(String url, String data) {
         mIsLoaded = true;
     }
 
+    public void setOriginAccessWhitelist(String url, String[] patterns) {
+        if (mNativeContent == 0 ||
+                (url == null || url.isEmpty())) {
+            return;
+        }
+
+        // Reset origin access whitelists if pattern is empty.
```

Comment:
Yes, we can reset origin access whitelists with null. it will be refined to
// Reset origin access whitelists if pattern is null